api/src/config/index.ts

```diff
@@ -129,9 +129,11 @@ export const config: Config = {
     level: 'verbose',
   },
   cache: {
-    type: 'memory',
+    type: process.env.REDIS_ENABLED === 'true' ? 'redis' : 'memory',
     ttl: 60 * 1000, // Milliseconds
     max: 100, // Maximum number of items in cache (defaults to 100)
+    host: process.env.REDIS_HOST,
```

Comment:
```suggestion
    host: process.env.REDIS_HOST,
```
Let's add a default value (same as we have in our docker-compose.redis.yml)